images/router/haproxy/conf/haproxy-config.template

```diff
@@ -302,7 +306,11 @@ backend be_tcp_{{$cfgIdx}}
     {{ range $serviceUnitName, $weight := $cfg.ServiceUnitNames }}
       {{ with $serviceUnit := index $.ServiceUnits $serviceUnitName }}
         {{ range $idx, $endpoint := endpointsForAlias $cfg $serviceUnit }}
-  server {{$endpoint.ID}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter {{env "ROUTER_BACKEND_CHECK_INTERVAL" "5000ms"}} weight {{ $weight }}
+            {{ with $healthIntv := index $cfg.Annotations "router.openshift.io/haproxy.health.check.interval" }}
+  server {{$endpoint.IdHash}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter {{ $healthIntv }} weight {{ $weight }}
+            {{ else }}
+  server {{$endpoint.IdHash}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter 5000ms weight {{ $weight }}
```

Comment:
@sferich888  yeah, as @knobunc mentioned having a global setting (via an env variable) with a route specific override (via an annotation) probably gives us the maximum configuration flexibility.  Plus validate the `healthIntv` value - something similar to https://github.com/openshift/origin/blob/master/images/router/haproxy/conf/haproxy-config.template#L296

We might need a few `else` parts to default to using the environment variable/5000 ms fallback.